meson_options.txt

```diff
@@ -1,5 +1,6 @@
 option('audit', type: 'boolean', value: false, description: 'Audit support')
 option('launcher', type: 'boolean', value: true, description: 'Build compatibility launcher')
+option('docs', type: 'boolean', value: false, description: 'Build documentation')
```

Comment:
Would't it be better to have a `true` default value, at least for compatibility with the old build defaults?

Reply:
Cannot speak for Tom here, but our preference is generally to default to 'false'. I know we didn't follow this with the 'launcher' option, but maybe we should?

We also document those changes in 'NEWS', so package maintainers can adapt to it. I understand, though, that we should keep burden on them as small as possible.

Tom, any specific reason to default to false, other than that it feels right that way?

Reply:
Sorry for the late reply.

My reasoning was the same as David's, we should default to false for anything that isn't strictly speaking necessary on all systems.

As for the launcher, I'm fine with that being enabled by default, at least until we document the broker API, declare it stable and someone actually uses it directly (making it usable without the launcher).

Reply:
No strong preference. I was pointing this out in the case that there were no strong reasons behind the choice, but the justifications to default to `false` is sound.